test/e2e/webdriver/chrome.js

```diff
@@ -38,7 +38,7 @@ class ChromeDriver {
       args.push(`load-extension=${process.cwd()}/dist/chrome`);
     }
 
-    if (openDevToolsForTabs) {
+    if (responsive) {
```

Comment:
Thoughts on leaving both options, so that we can still enable the use of dev tools in tests that don't require a small viewport?

```suggestion
    // When "responsive" is enabled, open dev tools to force a smaller viewport
    // The minimum window width on Chrome is too large, this is how we're forcing the viewport to be smaller
    if (openDevToolsForTabs  || responsive) {
```